Examples/UIExplorer/StatusBarExample.js

```diff
@@ -0,0 +1,181 @@
+/**
+ * The examples provided by Facebook are for non-commercial testing and
+ * evaluation purposes only.
+ *
+ * Facebook reserves all rights not expressly granted.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS
+ * OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON INFRINGEMENT. IN NO EVENT SHALL
+ * FACEBOOK BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN
+ * AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+ * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ *
+ * @flow
+ */
+'use strict';
+
+const React = require('react-native');
+const {
+  StyleSheet,
+  View,
+  Text,
+  TouchableHighlight,
+  StatusBar,
+} = React;
+
+type BarStyle = 'default' | 'light-content';
+
+type State = {
+  animated: boolean,
+  color: string,
+  hidden?: boolean,
+  translucent?: boolean,
+  barStyle?: BarStyle,
+  networkActivityIndicatorVisible?: boolean
+};
+
+const colors = [
+  '#ff0000',
+  '#00ff00',
+  '#0000ff',
+];
+
+const barStyles = [
+  'default',
+  'light-content',
+];
+
+exports.framework = 'React';
+exports.title = '<StatusBar>';
+exports.description = 'Component for controlling the status bar';
+
+const StatusBarExample = React.createClass({
+  getInitialState(): State {
+    return {
+      animated: true,
+      color: this._getColor(0),
+    };
+  },
+
+  _colorIndex: 0,
+  _barStyleIndex: 0,
+
+  _getColor(index: number) {
+    return colors[index % colors.length];
+  },
+
+  _getBarStyle(index: number): BarStyle {
+    return barStyles[index % barStyles.length];
+  },
+
+  render() {
+    return (
+      <View>
+        <StatusBar
+          color={this.state.color}
+          translucent={this.state.translucent}
+          hidden={this.state.hidden}
+          animated={this.state.animated}
+          barStyle={this.state.barStyle}
+          networkActivityIndicatorVisible={this.state.networkActivityIndicatorVisible}
+        />
+        <View>
+          <TouchableHighlight
+            style={styles.wrapper}
+            onPress={() => this.setState({animated: !this.state.animated})}>
+            <View style={styles.button}>
+              <Text>Set animated `{!this.state.animated ? 'true' : 'false'}`</Text>
```

Comment:
I found this example a bit confusing. When a button is labels "Set animated true", it's not entirely obvious that the current state is "false" and that the button will set it to "true" - instead it looks like maybe "true" is the current state.

I'd suggest making this unambiguous by changing the format of the button labels to:

    animated: true
    style: default
    etc...

That way the label reflects the current state.